libs/util/src/lib/generateUid.ts

```diff
@@ -0,0 +1,22 @@
+import { AppIdNumberType } from '@razor/models';
+import { customAlphabet } from 'nanoid';
+
+const nanoid = customAlphabet(
+  'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789',
+  10,
+);
+
+const tournamentIdLength = 8;
+const playerIdLength = 8;
+const generalIdLength = 8;
+
+export const generateUid = async (type: AppIdNumberType): Promise<string> => {
```

Comment:
have a dynamic return type based on the `type`

Reply:
Solved in PR #14 under commit 0fbe5ff4d06d17be7e2cfbbc1cd667bd024e6cca